pkg/config/app.go

```diff
@@ -10,8 +10,8 @@ var (
 )
 
 func Connect() {
-	// dsn := "root:password@tcp(127.0.0.1:3306)/karygo?charset=utf8mb4&parseTime=True&loc=Local"
-	dsn := "ba15d8eef097fd:bc99a3bc@tcp(us-cdbr-east-06.cleardb.net:3306)/heroku_60b5ede111b594a?charset=utf8mb4&parseTime=True&loc=Local"
+	dsn := "root@tcp(127.0.0.1:3306)/karygo?charset=utf8mb4&parseTime=True&loc=Local"
```

Comment:
when pushing switch back to the heroku database config, else it will break online

Reply:
Noted